auth0_flutter_platform_interface/lib/src/method_channel_auth0_flutter_auth.dart

```diff
@@ -52,6 +57,39 @@ class MethodChannelAuth0FlutterAuth extends Auth0FlutterAuthPlatform {
     return Challenge.fromMap(result);
   }
 
+
+  @override
+  Future<void> passwordlessWithEmail(
+      final ApiRequest<AuthPasswordlessLoginOptions> request) async {
+     await invokeRequest(method: authPasswordlessWithEmailMethod,
+         request: request);
+  }
+
+
+  @override
+  Future<Credentials> loginWithEmail(
```

Comment:
This one can be confusing, given that unlike in Swift, the method is just called `loginWithEmail (in Swift the param names are part of the method name). That seems like regular login with email/password.

Maybe call it `loginWithEmailCode`?